Azure/sonic-utilities
# Add check to not allow deleting PO if its member of vlan.

#### What I did
Added check to not allow deleting PortChannel if its member of vlan.
#### How I did it
Added a check in the command script.
#### How to verify it
1. Create a portchannel.
2. Add the portchannel to a vlan.
3. Try to delete the portchannel.
4. It should throw an error message.
#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
Error: PortChannel1 has vlan Vlan100 configured, remove vlan membership to proceed
